backend/tracim_backend/lib/mail_notifier/notifier.py

```diff
@@ -351,12 +353,13 @@ def notify_created_account(self, user: User, password: str) -> None:
         subject = translated_subject.replace(EST.WEBSITE_TITLE, str(self.config.WEBSITE__TITLE))
         message = MIMEMultipart("alternative")
         message["Subject"] = subject
-        message["From"] = self._get_sender()
+        message["From"] = self._get_sender(origin_user)
```

Comment:
In origin_user, you may have the real user (with its specific email), no?

Reply:
The code is not safe, I think: if you call this method with a wrong user instance, you will try to send email with a from you are not authorized to.

Reply:
After discussion, I'm wrong: `self._get_sender(origin_user)` builds a "from" like `Damien via Tracim<notification+4@tracim.fr>`. Probably renaming this method would be a way not to misunderstand what the code does